cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,12 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
-
+    cy.visit('https://react-redux.realworld.io');
+    cy.get(':nth-child(2) > .nav-link').click();
+    cy.get(':nth-child(1) > .form-control').type('tester111@test.net');
+    cy.get(':nth-child(2) > .form-control').type('tester111');
```

Comment:
Similar to the above point, it's not recommended to use CSS selector like ':nth-child(2) > .form-control'. Use a more specific class or id selector, or data-test-id attribute.